docs/index.rst

```diff
@@ -98,7 +99,7 @@ Table of Contents
 
    source/features/hydra
    source/features/multi_gpu
-   source/features/tiled_rendering
+   Tiled Rendering</source/overview/sensors/camera>
```

Comment:
This looks like it's wrong

Reply:
it LOOKS wrong, but it isn't :) I think the toctree expects single line entries (because rst has all kinds of whitespace rules). this is how you link to a doc but have the link named something other than the doc name.  in this case, the Tiled Rendering feature page was absorbed into the camera doc, but "Cameras" isn't the name of the feature we want in the menu, it's "Tiled Rendering"

![image](https://github.com/user-attachments/assets/439e5e72-2b0e-4fea-ab96-9da55347872b)